src/plone/testing/z2.py

```diff
@@ -543,6 +543,8 @@ def setUpDatabase(self):
         self['zodbDB'] = zodb.stackDemoStorage(
             self.get('zodbDB'),
             name='Startup')
+        import transaction
```

Comment:
Ok, I just followed the pattern used at other places in the file, I think hence this is not test code but test setup code import in the header is fine. I will change it and remove also the other occurrences.